ClientApp/src/app/flights-search-bar/flights-search-bar.component.ts

```diff
@@ -29,68 +30,86 @@ export class FlightsSearchBarComponent implements OnInit, AfterViewChecked{
     ticketsNumber: new FormControl(1, Validators.min(1))
   });
 
+  airports: Airport[];
   airports$: Observable<Airport[]>;
   private searchTerms = new Subject<string>();
 
   today = new Date();
+  maxDate = new Date(new Date(this.today).getTime() + 1000 * 60 * 60 * 24 * 180);
```

Comment:
What is 1000 * 60 * 60 * 24 * 180?

Seems like you have to add 180 days. There is a method from Date.prototype.setDate() [check here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate) in MDN

```ts
maxDate = new Date(); 
dateInterval = 180;

this.maxDate.setDate(this.today.getDate() + dateInterval ); // in constructor or ngOnInit I guess
```